src/hotspot/share/gc/shenandoah/shenandoahOldGeneration.cpp

```diff
@@ -0,0 +1,30 @@
+#include "precompiled.hpp"
```

Comment:
Missing header.

Reply:
Wow, I sure did miss a lot of copyright headers. Can `git jcheck` be made to check for this?

Reply:
`grep -r -L -i copyright src/hotspot/share/gc/shenandoah` also works.